src/codegen/sys/statics.rs

```diff
@@ -19,7 +19,7 @@ pub fn after_extern_crates(w: &mut dyn Write) -> Result<()> {
         "#[allow(unused_imports)]",
         "use libc::{c_int, c_char, c_uchar, c_float, c_uint, c_double,",
         "    c_short, c_ushort, c_long, c_ulong,",
-        "    c_void, size_t, ssize_t, intptr_t, uintptr_t, FILE};",
+        "    c_void, size_t, ssize_t, intptr_t, uintptr_t, FILE, in6_addr, stat};",
```

Comment:
`in6_addr` doesn't exist on windows' libc. You need to add it there first.

Reply:
You mean add it to rust libc library ?

Reply:
Yes, I'm creating a PR for it because it has unions which is a bit tricky.

Reply:
Thanks!

Reply:
I opened https://github.com/rust-lang/libc/pull/2741.

Reply:
Do you know how much time usually those PR takes to land ?

Reply:
Considering it's a small one, I think a few days.